blog/2023-08-18-network.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Network Team Update
+slug: 2023-08-18-network
+authors: coot
+tags: [network]
+hide_table_of_contents: false
+---
+
+## High-level overview of [sprint 42][sprint-42]
+
+### Eclipse Evasion
+
+We merged and released a new version of the `ouroboros-network` package (version
+`0.9.0.0`) which includes [big ledger peers feature][ouroboros-network#3886].
+This is the primary mechanism to defend against eclipses.   We also prepared
+a PR to updated [`ouroboros-cosnensus`][ouroboros-consensus#165] and
+[`ekg-forward`][ekg-forward#30] packages.
+
+### CDDL specs for protocol codecs
+
+We made our `cddl` spec for network codec more inlined with the
+implementation which is highly polymorphic.  `cddl` doesn't have the notion of
+polymorphism, but has `any` which can generate any valid cbor term.  We matched
+it with an `Any` type on the Haskell side and made all remaining tests & specs
+use it.  This simplified the specifications and made it easier to understand
+which parts are defined in the spec, and which parts are left unspecified.
+See [ouroboros-network#4595].
+
+### Ouroboros-Network-Framework API changes
+
+We also released `ouroboros-network-framework` and other network components.
+The `ouroboros-network-framework` package contains a redesign of API exposed to
+`ouroboros-cosnensus`.  We consolidated, cleaned it and made it easier to
+extend in the future if there will be new arguments that need to be passed to
+mini-protocol initiator and responders which comes from the low-level network
+layer.
+
+### Nix setup (CI)
+
+We also made a major review of our `nix` setup.  With help from our DevX team
+we ended up with a clean `flake.nix` file which can:
+
+* compile & test the code on `x86_64-linux`, `x86_64-darwin` and `aarch64-darwin`
+* cross-compile to `Windows` on `x86_64-linux` 
+
+And provides a shell which contains all the build tools, including `ghc-9.6`,
+`hls`, `cddl`, and more.  See [ouroboros-network#4640],
+[ouroboros-network#4643].
+
+
+## Other contributions
+
+### Cardano Network Service Assurance
+
+* The busywork and writeup in finishing up the CNSA, first stage (first
```

Comment:
do you really mean 'busywork' here - that is usually a term for work not needed to be done , I don't think you intended the derogatory meaning here! :)

Reply:
Oh, I didn't know, I'll remove it. I got this part from Mark, maybe it's UK / US difference?